quickstart.mdx

```diff
@@ -55,24 +67,32 @@ If you prefer to work locally, follow these steps to get started.
       ```python Python
       from tilebox.workflows import Client, Task
 
-      class MyFirstTask(Task):
-        def execute(self):
-          print("Hello World from my first Tilebox task!")
-
+      # Replace with your actual cluster and token
+      cluster = "YOUR_COMPUTE_CLUSTER"
       client = Client(token="YOUR_TILEBOX_API_KEY")
 
-      # submit a job
+      class HelloWorldTask(Task):
+          greeting: str = "Hello"
+          name: str = "World"
+
+          def execute(self, context):
+              print(f"{self.greeting} {self.name}, from the main task!")
+              context.submit_subtask(HelloSubtask(name=self.name), cluster=cluster, depends_on=None)
```

Comment:
This is the same as
`context.submit_subtask(HelloSubtask(name=self.name))` - should we go for the simple shorter version for the quickstart?

Reply:
Yeah we can do that. The reason why I put it is so that people can see that they _could_ set dependencies or different clusters. But maybe there's a better place for that somewhere

Reply:
Haha yeah I mentioned this because this is exactly the difficult thing with such pages. In this case the goal of this page is to just get started quickly - and not to act as a full reference of what's possible, so I think it makes sense to show the simple example :smile: